README.md

```diff
@@ -1,2 +1,33 @@
-# peer-server
+# Peer-server
+
 Node server for Peer
+
+## Scripts
+
+> yarn run build
+
+Compiles all the typescript files down to javascript. Typescript is supposed to compile down to javascript, and the application is supposed to take those js files and run them. Run this command if you've made changes to any ts files and need to recompile the project so `dist` has the latest changes. (The compiled ts files will be in a folder called `dist` and it will automatically be created for you if you're running this command for the first time.)
```

Comment:
We probably don't need the explanation - i would remove the second sentence and add "and runs it" to the first sentence.

Reply:
I wrote it in for Julio since he's new to TypeScript, but I can remove it! Lmk! @aej11a

Reply:
Doesn't matter to me too much!

Reply:
Cool, I'll keep it in for now, and we can remove it later!